packages/pigeon/README.md

```diff
@@ -1,7 +1,7 @@
 # Pigeon
 
 Pigeon is a code generator tool to make communication between Flutter and the
-host platform type-safe, easier and faster.
+host platform type-safe, easier and faster to implement.
```

Comment:
While pigeon right now runs at the same speed and platform channels.  We are about to land a change that makes them faster.  That was always the goal.  Once we can abstract away the medium, we can start making improvements.

Reply:
I understand it is the goal. But the main problem why I want this text to change is I spent one working day implementing pigeon to make my code run faster and after testing if it is faster and how much to realize it is the same speed :(  So I want it to be clear for others too.
So should i just delete the whole word "faster"  for now?

But I am looking forward for the faster version 💯

Reply:
Sorry, since this was authored now for the same feature set, pigeon is faster than MethodChannels.  I don't think it's misleading to say that it is faster.

Reply:
(note too that it was always faster than method channels because of one less layer of indirection that results from multiplexing the channels)

Reply:
"MethodChannels" doesn't appear in the sentence anywhere. Someone could easily assume that the comparison is with message channels, not method channels, and it certainly isn't faster that the thing it is implemented using.